utl/flatbuffer/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
It looks like `utl/flatbuffer` directory should be under `vowpalwabbit/parser/flatbuffer/utl` with the other flatbuffer files. In that case you can generate the below executables in the `parser/flatbuffer`'s CMakeLists.txt.